### ISSUE TITLE
Only the last 500 calls are backed up

### ISSUE THREAD

**Issue by Woi** [jberkel/sms-backup-plus#840]
I observed that version 1.5.11-BETA12 only backed up the last 500 call logs, while I have more on my phone. Considering that there is a setting for the amount of SMS messages being backed up, which defaults to 500: Could it be that there is the same default value for call logs, but no menu to change it?

**Comment by jberkel** [comment]
Yes, this was changed as part of #586. For call logs, I need to check.

**Comment by jberkel** [comment]
Actually, #586 is about the restore count, so it doesn't apply.

**Comment by Woi** [comment]
I have some additional information now:
New calls are backed up as expected, giving me ~520 calls in my IMAP folder. Also I was able to restore all of them (checked newest, oldest and some random samples) , so I confirm it's not the restore count but backup up count only.

**Comment by jberkel** [comment]
@Woi the setting is called "Items per backup" and applies to all item types (added together). The default should be "All".

![screenshot_1522264242](https://user-images.githubusercontent.com/18891/38050805-82335e9e-32cc-11e8-9fad-f9434639f0a8.png)

**Comment by Woi** [comment]
It was set to "All", since SMS were completely backed up.
